conf/dbs.config

```diff
@@ -0,0 +1,49 @@
+/*
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Nextflow config file for databases links
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+    Defines pointers to the DBS publicly available that store models parametrisations
+    and data.
+----------------------------------------------------------------------------------------
+*/
+params {
+
+    // AlphaFold2 links
+    bfd               = 'https://storage.googleapis.com/alphafold-databases/casp14_versions/bfd_metaclust_clu_complete_id30_c90_final_seq.sorted_opt.tar.gz'
+    small_bfd         = 'https://storage.googleapis.com/alphafold-databases/reduced_dbs/bfd-first_non_consensus_sequences.fasta.gz'
+    alphafold2_params = 'https://storage.googleapis.com/alphafold/alphafold_params_2022-03-02.tar'
+    mgnify            = 'https://storage.googleapis.com/alphafold-databases/casp14_versions/mgy_clusters_2018_12.fa.gz'
+    pdb70             = 'http://wwwuser.gwdg.de/~compbiol/data/hhsuite/databases/hhsuite_dbs/old-releases/pdb70_from_mmcif_200916.tar.gz'
+    pdb_mmcif         = 'rsync.rcsb.org::ftp_data/structures/divided/mmCIF/' //'rsync.rcsb.org::ftp_data/structures/divided/mmCIF/' ftp.pdbj.org::ftp_data/structures/divided/mmCIF/ rsync.ebi.ac.uk::pub/databases/pdb/data/structures/divided/mmCIF/
+    pdb_obsolete      = 'ftp://ftp.wwpdb.org/pub/pdb/data/status/obsolete.dat'
+    uniclust30        = 'https://storage.googleapis.com/alphafold-databases/casp14_versions/uniclust30_2018_08_hhsuite.tar.gz'
+    uniref90          = 'ftp://ftp.uniprot.org/pub/databases/uniprot/uniref/uniref90/uniref90.fasta.gz'
+    pdb_seqres        = 'ftp://ftp.wwpdb.org/pub/pdb/derived_data/pdb_seqres.txt'
+    uniprot_sprot     = 'ftp://ftp.ebi.ac.uk/pub/databases/uniprot/current_release/knowledgebase/complete/uniprot_sprot.fasta.gz'
+    uniprot_trembl    = 'ftp://ftp.ebi.ac.uk/pub/databases/uniprot/current_release/knowledgebase/complete/uniprot_trembl.fasta.gz'
```

Comment:
All of theses are URLs and FTP links.
Any other possibilities?

Reply:
All these URLs and FTP links are needed, when the user wants from the pipeline to download automatically the dbs and model parameters required by AlphaFold2 and Colabfold. We were thinking with @JoseEspinosa of alternatives, but we ended up that this solution (putting them isolated from the rest of the pipeline params) might be the best, because these links are upon changes in the future, when newer versions will come out. But any suggestions are more than welcome!